requirements.txt

```diff
@@ -0,0 +1,29 @@
+botocore==1.31.4
```

Comment:
What's the use case for creating the requirements file instead of keeping those dependencies in the environment file itself?

Reply:
Devs/users can run `pip install -r requirements.txt` easier instead of extracting from the file. Conda will still build the same way.